src/ResourceManagement/HDInsight/HDInsight/Microsoft.Azure.Management.HDInsight.nuget.proj

```diff
@@ -2,8 +2,8 @@
 <Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <ItemGroup>
     <SdkNuGetPackage Include="Microsoft.Azure.Management.HDInsight">
-      <PackageVersion>1.3.0</PackageVersion>
+      <PackageVersion>2.0.0</PackageVersion>
```

Comment:
@markcowl:  The old properties are disabled; they are marked obsolete and an error is given if used -- @srsiva and/or @shefaliv would need to confirm.

We wanted to do this since it provides a more actionable error message than simply removing the old fields. It guides the user towards the new usage.

If we should remove the old fields instead of doing this, let us know and we can do that. We just wanted to give users a version in which the old fields would still exist, just be marked 'obsolete' and disabled.